COLLABORATOR_GUIDE.md

```diff
@@ -504,12 +505,16 @@ The TSC should serve as the final arbiter where required.
 
 ## Landing Pull Requests
 
+1. Avoid landing PRs that are assigned to someone else. Authors who wish to land
+   their own PRs will self-assign them, or delegate to someone else. If in
+   doubt, ask the assignee whether it is okay to land?
 1. Never use GitHub's green ["Merge Pull Request"][] button. Reasons for not
    using the web interface button:
-   * The merge method will add an unnecessary merge commit.
-   * The squash & merge method can add metadata (the PR #) to the commit title.
-   * If more than one author has contributed to the PR, keep the most recent
-     author when squashing.
+   * The "Create a merge commit" method will add an unnecessary merge commit.
+   * The "Squash and merge" method will add metadata (the PR #) to the commit
+     title, and if more than one author has contributed to the PR, squashing
+     will only keep the most recent author when squashing.
```

Comment:
Maybe this?:

> The "Squash and merge" method will add metadata (the PR #) to the commit     title. If more than one author has contributed to the PR, squashing will only keep the most recent author.

If you don't like that and prefer the existing wording, then my comment is to please remove `when squashing` in `squashing will only keep the most recent author when squashing.`

Reply:
Ack